src/H5.c

```diff
@@ -57,6 +57,10 @@ static int H5_mpi_delete_cb(MPI_Comm comm, int keyval, void *attr_val, int *flag
 /* Library Private Variables */
 /*****************************/
 
+/* Library incompatible release versions */
+const unsigned VERS_RELEASE_EXCEPTIONS[]    = {0};
+const unsigned VERS_RELEASE_EXCEPTIONS_SIZE = 0;
```

Comment:
if release 0 is an exception, shouldn't size be 1?  or else list of exceptions should be {}?

Reply:
The SIZE is what matters, the list is invalid. Same as 1.12 so it is consistent.